tests/conftest.py

```diff
@@ -2,6 +2,8 @@
 
 import pytest
 import tables
+from _pytest.logging import LogCaptureFixture
```

Comment:
What is `_pytest`?

Reply:
Cargo culting.  This is a pattern from the loguru docs for making pytest work correctly with loguru.